v2/datastream-to-spanner/src/main/java/com/google/cloud/teleport/v2/templates/DataStreamToSpanner.java

```diff
@@ -712,16 +713,20 @@ public static PipelineResult run(Options options) {
     // of building pieces of the DLQ.
     PCollectionTuple reconsumedElements = null;
     boolean isRegularMode = "regular".equals(options.getRunMode());
-    if (isRegularMode && (!Strings.isNullOrEmpty(options.getDlqGcsPubSubSubscription()))) {
+    List<String> filePathsToIgnore =
+        new ArrayList<>(Arrays.asList("/tmp_retry", "/tmp_severe/", ".temp"));
+    if (isRegularMode) {
+      filePathsToIgnore.add("/severe/");
+    } else {
+      filePathsToIgnore.add("/retry/");
```

Comment:
In the retryMode, why is the retryBucket being ignored ?
If there is a temporary failure, will that not be retried ? Also, if there is a severe error, will it be retried multiple times ?